src/main/java/com/lmax/disruptor/EventProcessor.java

```diff
@@ -38,5 +38,9 @@
      */
     void halt();
 
+    /**
+     * @return whether this event processor is running or not
+     * Implementations should ideally return true only when the associated thread is idle.
```

Comment:
> Implementations should ideally return true only when the associated thread is idle

I thought `isRunning` returns true only when the associated thread is _NOT_ idle?

Reply:
booleans will be booleans. Good catch, thank you.